cmake/platform/windows.cmake

```diff
@@ -130,20 +130,23 @@ function(add_python_executable)
   endif()
 
   if(WIN32)
+    get_filename_component(WRAPPER_NAME
+      "${ARG_TARGET_NAME}"
+      NAME_WLE)
```

Comment:
This option is only available as of CMake 3.14. I suggest to use `NAME_WE` instead.

Reply:
Done. I switched to use the `NAME_WE`.